### ISSUE TITLE
open source code

### ISSUE THREAD

**Issue by OliverHuang1220** [youngfly11/ReIR-WeaklyGrounding.pytorch#2]
Hi @youngfly11
Thank you for your excellent work! I'm very interested in your work. Will you open source code？

**Comment by youngfly11** [comment]
We will release the code. But recently I am not in campus because of the internship. Most of the codes and data are in the school server.  But I can provide the loss function scripts for you to implement the loss we design. 
When I return to the campus (In September), I will release the code intermediately.

**Comment by OliverHuang1220** [comment]
Thank you for your reply. I can learn more about your paper if you provide loss function.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Yongfei ***@***.***&gt;; 
发送时间: 2021年8月5日(星期四) 下午3:51
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [youngfly11/ReIR-WeaklyGrounding.pytorch] open source code (#2)





 
We will release the code. But recently I am not in campus because of the internship. Most of the codes and data are in the school server.  But I can provide the loss function scripts for you to implement the loss we design.
 When I return to the campus (In September), I will release the code intermediately.
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by yechenzhi** [comment]
hi, I bet you know what I'm gonna ask.....could you please release your code, I'm so vegetable.

**Comment by yechenzhi** [comment]
It's already October

**Comment by youngfly11** [comment]
Hi, yechenzhi;
I am sorry for the late update.  Currently, most codes are released. There are still some details to be further improved, like the dataset preprocessing.  I will continue to update in the next few days.

Yongfei

**Comment by JJ-res101** [comment]
> Hi, yechenzhi; I am sorry for the late update. Currently, most codes are released. There are still some details to be further improved, like the dataset preprocessing. I will continue to update in the next few days.
> 
> Yongfei

hi, youngfly:
Could you provide the dataset preprocessing scripts now? And I wonder, is the current code easy to run without them？Thanks!

**Comment by youngfly11** [comment]
Hi; JJ-res101;

The Bottom-up-attention weight can be found at https://github.com/violetteshev/bottom-up-features;
the preprocessed scene graph annotation & sentence annotations can be found at https://github.com/youngfly11/LCMCG-PyTorch (google drive contains some necessary files about flickr30k dataset)
